packages/tracing-internal/src/browser/browsertracing.ts

```diff
@@ -392,10 +415,21 @@ export class BrowserTracing implements Integration {
         return undefined;
       }
 
-      if (inflightInteractionTransaction) {
-        inflightInteractionTransaction.setFinishReason('interactionInterrupted');
+      if (op === 'ui.action.left.mouseup' && inflightInteractionTransaction) {
+        inflightInteractionTransaction.startChild({
+          description: htmlTreeAsString(target),
+          op,
+          startTimestamp: timestampInSeconds(),
+          endTimestamp: timestampInSeconds() + 0.001,
+        });
```

Comment:
Ideally we can get the timestamps from performance entries, but for now i'm just marking this in the event handler.

Reply:
do we also need to set a finish reason here?